packages/@uppy/locales/src/de_DE.js

```diff
@@ -1,150 +1,184 @@
   importFrom: 'Importieren von %{name}',
+  inferiorSize: 'Diese Datei ist kleiner als die minimal erlaubte Dateigröße von %{size}',
   loading: 'Laden...',
-  logOut: 'Ausloggen',
+  logOut: 'Abmelden',
+  micDisabled: 'Zugriff auf Mikrofon von Benutzer abgelehnt',
+  missingRequiredMetaField: 'Fehlende erforderliche Meta-Felder',
+  missingRequiredMetaFieldOnFile: 'Fehlende erforderliche Meta-Felder in %{fileName}',
   myDevice: 'Mein Gerät',
   noCameraDescription: 'Bitte Kamera anschließen, um Bilder oder Videos aufzunehmen',
   noCameraTitle: 'Kamera nicht verfügbar',
   noDuplicates: 'Datei \'%{fileName}\' existiert bereits und kann nicht erneut hinzugefügt werden',
-  noFilesFound: 'Sie haben keine Dateien oder Ordner hier',
+  noFilesFound: 'Sie haben hier keine Dateien oder Ordner',
   noInternetConnection: 'Keine Internetverbindung',
   noNewAlreadyUploading: 'Während der Upload läuft, können keine weiteren Dateien hinzugefügt werden',
+  openFolderNamed: 'Ordner %{name} öffnen',
   pause: 'Pausieren',
-  pauseUpload: 'Upload pausieren',
   paused: 'Pausiert',
-  poweredBy: 'Angetrieben von %{uppy}',
+  pauseUpload: 'Hochladen pausieren',
+  pluginNameBox: 'Box',
+  pluginNameCamera: 'Kamera',
+  pluginNameDropbox: 'Dropbox',
+  pluginNameFacebook: 'Facebook',
+  pluginNameGoogleDrive: 'Google Drive',
+  pluginNameInstagram: 'Instagram',
+  pluginNameOneDrive: 'OneDrive',
+  pluginNameZoom: 'Zoom',
+  poweredBy: 'Powered by %{uppy}',
   processingXFiles: {
-    '0': '%{smart_count} Datei verarbeiten',
+    '0': 'Eine Datei verarbeiten',
     '1': '%{smart_count} Dateien verarbeiten',
-    '2': '%{smart_count} Dateien verarbeiten',
   },
+  recording: 'Aufnahme',
   recordingLength: 'Aufnahmedauer %{recording_length}',
+  recordingStoppedMaxSize: 'Die Aufnahme wurde gestoppt, weil die Dateigröße das Limit überschreitet',
+  recoveredAllFiles: 'Wir haben alle Dateien wiederhergestellt. Sie können mit dem Hochladen fortfahren.',
+  recoveredXFiles: {
+    '0': 'Wir konnten eine Datei nicht vollständig wiederherstellen. Bitte wählen Sie sie erneut aus und fahren Sie dann mit dem Hochladen fort.',
+    '1': 'Wir konnten %{smart_count} Dateien nicht vollständig wiederherstellen. Bitte wählen Sie sie erneut aus und fahren Sie dann mit dem Hochladen fort.',
+  },
   removeFile: 'Datei entfernen',
+  reSelect: 'Erneut auswählen',
   resetFilter: 'Filter zurücksetzen',
   resume: 'Fortsetzen',
-  resumeUpload: 'Upload fortsetzen',
+  resumeUpload: 'Hochladen fortsetzen',
   retry: 'Erneut versuchen',
-  retryUpload: 'Upload erneut versuchen',
+  retryUpload: 'Hochladen erneut versuchen',
+  revert: 'Rückgängig machen',
+  rotate: 'Drehen',
+  save: 'Speichern',
   saveChanges: 'Änderungen speichern',
+  searchImages: 'Suche nach Bildern',
   selectX: {
-    '0': 'Wählen Sie %{smart_count}',
-    '1': 'Wählen Sie %{smart_count}',
-    '2': 'Wählen Sie %{smart_count}',
+    '0': 'Wähle %{smart_count}',
+    '1': 'Wähle %{smart_count}',
```

Comment:
Why no Sie here?

Reply:
Wasn't sure where this is used in the app and thought without "Sie" would make more sense - going to revert it. Thanks!

Reply:
I don't know either :D Was just surprised by the change in style